### PR TITLE
Minor fixes to examples in torch models

### PR DESCRIPTION
**Pull request by arunppsg**
In this PR, I made a couple of minor changes to examples for models in torch models.

## Type of change

Please check the option that is related to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [X] Documentations (modification for documents)

## Checklist

- [X] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [X] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.22.0**)
  - [ ] Run `mypy -p deepchem` and check no errors
  - [X] Run `flake8 <modified file> --count` and check no errors
- [X] I have performed a self-review of my own code
- [X] New unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by rbharath — COMMENTED** [review]
The changes look good to me but the CI looks to be stuck with a lot of queued jobs (which I think are unrelated to this PR).

Can you run doctest locally on your machine to verify these all pass? I think you can simply do

```
python -m doctest filename.py
```
To run doctest (this might be good to add into our docs if it isn't documented already)

**Comment by arunppsg** [comment]
Have changes examples to use small sample data so that it could pass doctests.

File: deepchem/models/torch_models/attentivefp.py [hunk 1]
```diff
@@ -189,17 +189,18 @@ class AttentiveFPModel(TorchModel):
...
+  >>> # preparing dataset
+  >>> smiles = ["C1CCC1", "C1=CC=CN=C1"]
+  >>> labels = [0., 1.]
+  >>> featurizer = dc.feat.MolGraphConvFeaturizer(use_edges=True)
+  >>> X = featurizer.featurize(smiles)
+  >>> dataset = dc.data.NumpyDataset(X=X, y=labels)
+  >>> # training model
+  >>> model = AttentiveFPModel(mode='classification', n_tasks=1,
+  ...    batch_size=32, learning_rate=0.001)
+  >>> loss = model.fit(dataset, nb_epoch=50)
```
Comment (rbharath) [inline deepchem/models/torch_models/attentivefp.py:203]: Could you change this to nb_epoch=5? That will keep the runtime low

File: deepchem/models/torch_models/gcn.py [hunk 2]
```diff
@@ -211,17 +211,18 @@ class GCNModel(TorchModel):
...
+  >>> # preparing dataset
+  >>> smiles = ["C1CCC1", "CCC"]
+  >>> labels = [0., 1.]
+  >>> featurizer = dc.feat.MolGraphConvFeaturizer()
+  >>> X = featurizer.featurize(smiles)
+  >>> dataset = dc.data.NumpyDataset(X=X, y=labels)
+  >>> # training model
+  >>> model = GCNModel(mode='classification', n_tasks=1,
+  ...                  batch_size=32, learning_rate=0.001)
+  >>> loss = model.fit(dataset, nb_epoch=50)
```
Comment (rbharath) [inline deepchem/models/torch_models/gcn.py:225]: Change to 5 here as well

File: deepchem/models/torch_models/mpnn.py [hunk 3]
```diff
@@ -188,17 +188,18 @@ class MPNNModel(TorchModel):
...
+  >>> # preparing dataset
+  >>> smiles = ["C1CCC1", "CCC"]
+  >>> labels = [0., 1.]
+  >>> featurizer = dc.feat.MolGraphConvFeaturizer(use_edges=True)
+  >>> X = featurizer.featurize(smiles)
+  >>> dataset = dc.data.NumpyDataset(X=X, y=labels)
+  >>> # training model
+  >>> model = MPNNModel(mode='classification', n_tasks=1,
+  ...                  learning_rate=0.001)
+  >>> loss =  model.fit(dataset, nb_epoch=10)
```
Comment (rbharath) [inline deepchem/models/torch_models/mpnn.py:202]: Change to 5 here

File: deepchem/models/torch_models/torch_model.py [hunk 4]
```diff
@@ -47,12 +47,17 @@ class TorchModel(Model):
...
+  >>> import deepchem as dc
+  >>> import numpy as np
+  >>> X, y = np.random.random((10, 100)), np.random.random((10, 1))
+  >>> dataset = dc.data.NumpyDataset(X=X, y=y)
+  >>> pytorch_model = torch.nn.Sequential(
+  ...   torch.nn.Linear(100, 1000),
+  ...   torch.nn.Tanh(),
+  ...   torch.nn.Linear(1000, 1))
+  >>> model = dc.models.TorchModel(pytorch_model, loss=dc.models.losses.L2Loss())
+  >>> loss = model.fit(dataset, nb_epoch=10)
```
Comment (rbharath) [inline deepchem/models/torch_models/torch_model.py:60]: Change to 5 here

**Review by rbharath — COMMENTED** [review]
Looking good! One more set of minor changes and we should be able to merge

**Review by rbharath — APPROVED** [review]
The failures in doctest look to be unrelated to this PR so will go ahead and merge in!

**Pull request merged**